src/main/java/core/basesyntax/converter/TransactionParser.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.function.Function;
+
+public class TransactionParser implements Function<String, FruitTransaction> {
```

Comment:
Let's simplify it. You don't need to implement a functional interface, we use them for Stream/Optional API and use lambda expressions
```suggestion
public class TransactionParser {
```